cmd/eksctl/create.go

```diff
@@ -75,25 +76,27 @@ func createClusterCmd() *cobra.Command {
 	fs.IntVarP(&cfg.MinNodes, "nodes-min", "m", 0, "minimum nodes in ASG")
 	fs.IntVarP(&cfg.MaxNodes, "nodes-max", "M", 0, "maximum nodes in ASG")
 
+	fs.IntVar(&cfg.MaxPodsPerNode, "max-pods-per-node", 0, "maximum number of pods per node (set automatically if unspecified)")
 	fs.StringSliceVar(&availabilityZones, "zones", nil, "(auto-select if unspecified)")
 
+	fs.BoolVar(&cfg.NodeSSH, "ssh-access", false, "control SSH access for nodes")
 	fs.StringVar(&cfg.SSHPublicKeyPath, "ssh-public-key", DEFAULT_SSH_PUBLIC_KEY, "SSH public key to use for nodes (import from local path, or use existing EC2 key pair)")
 
 	fs.BoolVar(&writeKubeconfig, "write-kubeconfig", true, "toggle writing of kubeconfig")
 	fs.BoolVar(&autoKubeconfigPath, "auto-kubeconfig", false, fmt.Sprintf("save kubconfig file by cluster name, e.g. %q", kubeconfig.AutoPath(exampleClusterName)))
 	fs.StringVar(&kubeconfigPath, "kubeconfig", kubeconfig.DefaultPath, "path to write kubeconfig (incompatible with --auto-kubeconfig)")
 	fs.BoolVar(&setContext, "set-kubeconfig-context", true, "if true then current-context will be set in kubeconfig; if a context is already set then it will be overwritten")
 
-	fs.DurationVar(&cfg.WaitTimeout, "aws-api-timeout", eks.DefaultWaitTimeout, "")
+	fs.DurationVar(&cfg.WaitTimeout, "aws-api-timeout", api.DefaultWaitTimeout, "")
 	fs.MarkHidden("aws-api-timeout") // TODO deprecate in 0.2.0
-	fs.DurationVar(&cfg.WaitTimeout, "timeout", eks.DefaultWaitTimeout, "max wait time in any polling operations")
+	fs.DurationVar(&cfg.WaitTimeout, "timeout", api.DefaultWaitTimeout, "max wait time in any polling operations")
 
 	fs.BoolVar(&cfg.Addons.WithIAM.PolicyAmazonEC2ContainerRegistryPowerUser, "full-ecr-access", false, "enable full access to ECR")
```

Comment:
I think that flag would be mostly useful for debugging, or if you really have to hand-craft some feature that `eksctl` doesn't support yet. The main issue is that this code (by design) avoided use of parameters in CloudFormation as much as possible (e.g. subnets, node AMIs, SSH key name), and it's not unlikely even remaining parameters might go away, if there is a good reason for that. Any human readable template should have parameters, otherwise the human using it will try to add it in, and that would be a sad thing.

Reply:
I was thinking for the scenario where someone would want to use `eksctl` to generate the templates and use that template in a wider infrastructure provisioning script. But if this was the case then they could just exec eksctl directky (like we do with kops & terraform).

Perhaps for debugging we could have a hidden flag or even a log level that dumps the template (if thats not already done)?

Reply:
Providing (and documenting) an integration with Terraform would be 😍

Reply:
@mikemorris we aren't planning to integrate with Terraform, but we are keen to hear about your use-case – please open an issue and tell us about it 😄

Reply:
@errordeveloper Basically the same as https://github.com/kubernetes/kops/blob/master/docs/terraform.md - we're currently using kops to generate cluster spec and terraform to manage state. Considering migrating EKS and would love to stick to a similar workflow.

Reply:
@mikemorris - we use the same kops/terraform method and it would be good to do the same with EKS/terraform. If no one else looks at it then i will definetly have a look.

Reply:
Let's fork this thread into an issue :)

Reply:
#181